data-access/fractie.ts

```diff
@@ -7,18 +7,46 @@ import {
 } from 'mu';
 import { updateSudo } from '@lblod/mu-auth-sudo';
 
-import { getSparqlResults } from '../util/sparql-result';
+import { findFirstSparqlResult, getSparqlResults } from '../util/sparql-result';
 import { FRACTIE_TYPE } from '../util/constants';
-import { Term, TermProperty } from '../types';
 import { sparqlEscapeTermValue } from '../util/sparql-escape';
+import { Term, TermProperty } from '../types';
 
 export const fractie = {
+  areIdsValid,
   forBestuursperiode,
   addFractieOnPerson,
   addFractieOnPersonWithGraph,
   removeFractieWhenNoLidmaatschap,
 };
 
+async function areIdsValid(ids?: Array<string>): Promise<boolean> {
+  if (!ids || ids.length === 0) {
+    return false;
+  }
+
+  const values = ids.map((id) => sparqlEscapeString(id));
+  const countOfExisting = `
+    PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+    PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+
+    SELECT (COUNT(DISTINCT ?fractie ) as ?count)
+    WHERE {
+      VALUES ?fractieId { ${values.join('\n')} }
+        ?fractie a mandaat:Fractie.
```

Comment:
:ant: feels like we could extract this as a general helper function `areIdsValid(type:string, ids?: string[])`

Reply:
Done!